instant-xml/tests/entities.rs

```diff
@@ -10,32 +10,34 @@ struct StructSpecialEntities<'a> {
     string: String,
     str: &'a str,
     cow: Cow<'a, str>,
+    vec: Vec<String>,
 }
 
 #[test]
 fn escape_back() {
     assert_eq!(
         from_str(
-            "<StructSpecialEntities xmlns=\"URI\"><string>&lt;&gt;&amp;&quot;&apos;adsad&quot;</string><str>str</str><cow>str&amp;</cow></StructSpecialEntities>"
+            "<StructSpecialEntities xmlns=\"URI\"><string>&lt;&gt;&amp;&quot;&apos;adsad&quot;</string><str>str</str><cow>str&amp;</cow><list xmlns=\"\"><element xmlns=\"\">one</element><element xmlns=\"\">two</element><element xmlns=\"\">three</element></list></StructSpecialEntities>"
```

Comment:
I think we should use the field name and namespace as the list element tag (so `vec` in this case). I think `Vec`s will need a third variant in `Kind`. We should only have the `element` tag in case `T` is `Kind::Scalar`, otherwise we should use the nested type's element directly.

We definitely need tests for both a scalar `T` (which you already have) and a non-scalar one.